files/image_config/process-reboot-cause/process-reboot-cause

```diff
@@ -70,7 +70,7 @@ def main():
     # if there is no sonic_platform package installed, we only provide
     # software-related reboot causes.
     try:
-        import sonic_platform
+        import sonic_platform.platform
```

Comment:
@stephenxs: Why did you need to make this change? This is breaking support for other platforms? I plan to open a PR to revert this line, but I'd like to know your reasoning behind it.

Reply:
I've faced the same situation mentioned in [PR 3156](https://github.com/Azure/sonic-buildimage/pull/3156).
Having read it, I think your suggestion can work.

Reply:
Thanks for the reply. My PR to revert this line is here: https://github.com/Azure/sonic-buildimage/pull/3249

Reply:
Thank you. I'll update our code updated today.